src/Stratis.Bitcoin.Features.Apps/AppsStore.cs

```diff
@@ -28,38 +29,37 @@ public AppsStore(ILoggerFactory loggerFactory, IAppsFileService appsFileService,
 
         private void Load()
         {
-            IEnumerable<FileInfo> configs = this.appsFileService.GetStratisAppConfigFileInfos();
-            var apps = configs.Select(this.CreateApp).Where(x => x != null).ToList();
+            try
+            {
+                FileInfo[] fileInfos = new DirectoryInfo(this.dataFolder.ApplicationsPath)
+                    .GetFiles(ConfigFileName, SearchOption.AllDirectories);
+
+                IEnumerable<IStratisApp> apps = fileInfos.Select(x => new FileStorage<StratisApp>(x.DirectoryName))
+                                                         .Select(this.CreateAppInstance);
 
-            if (apps.IsEmpty())
+                this.applications.AddRange(apps.Where(x => x != null));
+
+                if (this.applications.IsEmpty())
+                    this.logger.LogWarning("No Stratis applications found at or below {0}",
+                        this.dataFolder.ApplicationsPath);
+            }
+            catch (Exception e)
             {
-                this.logger.LogWarning(
-                    $"No Stratis applications found at or below {this.appsFileService.StratisAppsFolderPath}");
-                return;
+                this.logger.LogError($"Failed to load Stratis apps : {0}", e.Message);                
```

Comment:
Don't use interpolated strings in logs - https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/using-logging.md#interpolated-strings